### DISCUSSION TITLE
Watch a public repo and run action

### DISCUSSION THREAD

**Discussion by NoelJacob** [community/community#48726] [Actions] [answered]
### Select Topic Area

Question

### Body

I would like to trigger an action only if an arbitrary public repo has a commit or a release made, etc.

**Comment by airtower-luna** [comment, marked as answer]
There's no event for things that happen in other repositories. I assume "arbitrary public repo" means that you can't get the other repo's maintainer to set up *sending* an event to yours, so the closest thing you can do is set up a [scheduled workflow](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) that checks the conditions you actually want. :thinking:
